qcodes/instrument_drivers/QuTech/IVVI.py

```diff
@@ -247,6 +309,43 @@ def ask(self, message, raw=False):
         message_len = self.write(message, raw=raw)
         return self.read(message_len=message_len)
 
+    def _read_raw_bytes_direct(self, size):
+        """ Read raw data using the visa lib """
+        with(self.visa_handle.ignore_warning(visa.constants.VI_SUCCESS_MAX_CNT)):
+            mes = self.visa_handle.visalib.read(self.visa_handle.session, size)
+
+        return mes[0]
```

Comment:
@peendebak  that it's hard to say why you return mes[0].
The most pythonic way of doing this would be:
```
a_nice_name_that_explains _what_this_variable_is = mes[0] 
```